package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/Enriquesuarezgtte/protractor-workshop-2018#readme",
   "dependencies": {
+    "jasmine-spec-reporter": "^4.2.1",
```

Comment:
is this a dependency or a dev dependency?

Reply:
my mistake. it should be a dev dependency